java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture2/tasks/_2/Number2.java

```diff
@@ -0,0 +1,229 @@
+            }
+            case "NORTH" -> {
+                System.out.println("Direction is NORTH\n");
+                robotGame.setDirection(Direction.NORTH);
+            }
+            case "WEST" -> {
+                System.out.println("Direction is WEST\n");
+                robotGame.setDirection(Direction.WEST);
+            }
+            case "EAST" -> {
+                System.out.println("Direction is EAST\n");
+                robotGame.setDirection(Direction.EAST);
+            }
+            default -> throw new InputMismatchException("Wrong input");
+        }
+
+        System.out.println("1.Move Forward.\n2.ChangeDirection;\n3.Get coordinates;\n4.Get directions;\n5.Exit");
+
+        while (!exit) {
+            switch (scanner.nextInt()) {
+                case 1:
+                    robotGame.moveForward();
+                    System.out.println("Moving On");
+                    break;
+                case 2:
+                    System.out.println("1.Turn Left;\n2.Turn Right;");
+                    if (scanner.nextInt() == 1) {
+                        robotGame.turnLeft();
+                        System.out.println("Turned left");
+
+                    }
+                    else  {
+                        robotGame.turnRight();
+                        System.out.println("Turned right");
+                    }
+                    break;
+                case 3:
+                    robotGame.getCoordinates();
+                    break;
+                case 4:
+                    robotGame.getDirection();
+                    break;
+                case 5:
+                    System.out.println("Shutting down");
+                    exit = true;
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+}
+
+enum Direction {
+    SOUTH,
+    NORTH,
+    WEST,
+    EAST
+}
+
+class RobotGame {
+
+    private int w;
+    private int h;
+    private int x;
+    private int y;
+    private int[] coordinates = {x, y};
+    int[][] roomSize;
+
+    private Direction direction = Direction.SOUTH;
+
+    public void setW(int w) {
+        this.w = w;
+    }
+
+    public void setH(int h) {
+        this.h = h;
+    }
+
+    public void getRoomSize() {
+        roomSize = new int[h][w];
+        for (int[] row : roomSize) {
+            for (int i : row) {
+                System.out.print(i);
+            }
+            System.out.println();
+        }
+    }
+
+
+    public void setDirection(Direction direction) {
+        this.direction = direction;
+    }
+
+    public void getCoordinates() {
+        System.out.println(Arrays.toString(coordinates));
+    }
+
+    public void getDirection() {
+        System.out.println(direction);
+    }
+
+    public void setCoordinates(int x, int y) {
+        this.x = x;
+        this.y = y;
+        this.coordinates = new int[]{x, y};
+    }
+
+    public void moveForward() {
```

Comment:
The task description also mentions Robot needs to have a method to move for N tiles